src/components/common/Toggle/index.js

```diff
@@ -0,0 +1,31 @@
+import React  from "react";
+import PropTypes from "prop-types";
+import * as Layout from "./style.module.css";
+import { useState } from "react";
+import {IoIosArrowUp, IoIosArrowDown} from "react-icons/io";
+
+const Toggle =(props)=>{
+	const [showDiv, setvisiblility] = useState(false);
```

Comment:
I want the parent to decide if we can show component or not. Right now all will be hidden by default